modules/gke-cluster/variables.tf

```diff
@@ -14,6 +14,21 @@
  * limitations under the License.
  */
 
+variable "backup_configs" {
+  description = "Configuration for Backup for GKE."
+  type = object({
+    enable_backup_agent = optional(bool, false)
+    backup_plans = optional(map(object({
+      region                            = string
+      schedule                          = string
+      retention_policy_days             = optional(string)
+      retention_policy_lock             = optional(bool, false)
+      retention_policy_delete_lock_days = optional(string)
+    })))
+  })
+  default = null
```

Comment:
Can we have here `default = {}` and then we can skip `try(...)` part when accessing this variable because with `{}` optional defaults will kick in.

Reply:
make sense, working on it (also fixing test and pushing again)